### ISSUE TITLE
Can not determine output type for

### ISSUE THREAD

**Issue by jeromeSH26** [MichalLytek/type-graphql#412]
Hi,

first of all, thks for this wonderful library, it is an amazing job, helping us to combine typescript, graphql and typeorm. Thks a lot

I'm facing an issue, that I canit figure out how to bypass.
I have 2 ObjectTypes with Fields of eachothers. The GoogleCalendarsResources object has a field eventsList that is an array of GoogleCalendarsEvents. Respictively, the  GoogleCalendarsEvents has a field that is an array of GoogleCalendarsResources .

When compiling, I got  the unable to determine the output type error for  the field listEvents.

I'm doing something wrong in the logic, but I don't know what is my design mistake.

Rgds

GoogleCalendarsResources.ts
```
import { GoogleCalendarsEvents } from "../../../../extendedTypesList";
@InterfaceType("IGoogleCalendarsResources")
abstract class IGoogleCalendarsResources {
    @Field(() => [GoogleCalendarsEvents], { nullable: true })
    eventsList: GoogleCalendarsEvents[];
}

@ObjectType({ implements: IGoogleCalendarsResources })
@InputType("GoogleCalendarsResourcesInputType")
class GoogleCalendarsResources implements IGoogleCalendarsResources {
    @Field(() => [GoogleCalendarsEvents], { nullable: true })
    eventsList: GoogleCalendarsEvents[];
}

export default GoogleCalendarsResources;
```

GoogleCalendarEvents.ts

```
@InterfaceType("IGoogleCalendarsEvents")
abstract class IGoogleCalendarsEvents {
    @Field(() => [GoogleCalendarsResources])
    resourcesList: GoogleCalendarsResources[];
}

@ObjectType({ implements: IGoogleCalendarsEvents })
@InputType("GoogleCalendarsEventsInputType")
class GoogleCalendarsEvents implements IGoogleCalendarsEvents {
      @Field(() => [GoogleCalendarsResources])
    resourcesList: GoogleCalendarsResources[];
}

export default GoogleCalendarsEvents;

```

**Comment by MichalLytek** [comment]
Unfortunately, I cannot reproduce the error 😕 

```ts
@Resolver()
class TestResolver {
  @Query(returns => GoogleCalendarsResources)
  resources() {
    return null as any;
  }

  @Query(returns => GoogleCalendarsEvents)
  events() {
    return null as any;
  }
}

async function main() {
  const schema = await buildSchema({
    resolvers: [TestResolver],
  });
  console.log(printSchema(schema));
}

main().catch(console.error);
```

```graphql
type GoogleCalendarsEvents implements IGoogleCalendarsEvents {
  resourcesList: [GoogleCalendarsResources!]!
}

input GoogleCalendarsEventsInputType {
  resourcesList: [GoogleCalendarsResourcesInputType!]!  
}

type GoogleCalendarsResources implements IGoogleCalendarsResources {
  eventsList: [GoogleCalendarsEvents!]
}

input GoogleCalendarsResourcesInputType {
  eventsList: [GoogleCalendarsEventsInputType!]
}

interface IGoogleCalendarsEvents {
  resourcesList: [GoogleCalendarsResources!]!
}

interface IGoogleCalendarsResources {
  eventsList: [GoogleCalendarsEvents!]
}

type Query {
  resources: GoogleCalendarsResources!
  events: GoogleCalendarsEvents!
}
```

Please provide a repository with a MINIMAL reproducible code example :wink:

**Comment by jeromeSH26** [comment]
OK I will try but this code is a small part of a big process.
Will still investigate on my side, because I have feeling that this is an import issue... 

Rgds

**Comment by jeromeSH26** [comment]
Dear,

Finally it was an issue when importing the types (I reall don't understand what is wrong). I had created a file that import all the types in order to get a single point to get the types. Using this file compiles fine, but I got the issue. Changing an importing the types straight from the file where it is created, and I haven't the issue.

So, nothing to do with type-graphql. Thks again for this great librairy. I'm in the process now to use dataloader and dependencies injections.
Rgds
